README.md

```diff
@@ -67,7 +68,7 @@ Read below to learn how to take part in improving Theia:
 
 ## Feedback
 
-Read below how to engage with Theia community:
+Read below how to engage with Eclipse Theia community:
```

Comment:
For the sake of simplicity, I'd prefer to leave this as is.

Reply:
done